### ISSUE TITLE
2 Questions when I try to run PASTA

### ISSUE THREAD

**Issue by zzzzzj52** [ai-med/PASTA#1]
I truly appreciate your excellent work! However, I encountered two issues while trying to rerun your work:

1. I keep getting an error, and as a beginner, I’m not quite sure how to resolve it.
2. I am uncertain about the meanings of the following variables: input_slice_channel, model_in_channels, and use_tabular_cond_encoder. How should I assign values to these variables?
![图片](https://github.com/user-attachments/assets/3df14ba5-4ec3-4133-ad1a-183003c6001a)

I would be extremely grateful for your response !
The followings are how i organize hdf5 with data demo and the configuration i set:

![图片](https://github.com/user-attachments/assets/6ee399f4-42d3-4ecf-8f55-a3040ec892cb)
##### Unet config ####

image_size: [113, 137]
model_in_channels: 15
out_channels_model: 15
unet_dim: 64
encoder_in_channels: 15
out_channels_encoder: 15
num_res_blocks: 4
attention_resolutions: [16, 8, 4]
num_heads: 4
unet_dim_mults: [1, 2, 3, 4]
dropout: 0
dims: 2
cond_emb_channels: 2048 # will always use condition unless it is null
resblock_updown: true
use_time_condition: true
tab_cond_dim: 9
use_tabular_cond_model: condition # condition
use_tabular_cond_encoder:  null # condition
tabular_cond: true
with_attention: True
cond_apply_method: 'ada'

##### Diffusion config ####

timestep_respacing: ddim100
beta_schedule: cosine
timesteps: 1000
objective: START_X
model_var_type: FIXED_LARGE
gen_type: ddim
condition: True
loss_type: l1
reconstructed_loss: l1
recon_weight: 0.1
rescale_intensity: True # rescale intensity to [-1, 1] during training

##### Trainer config ####

input_slice_channel: 15
data_dir: "/home/zhangzengji/Documents/PASTA/data_dir/MRI_PET7/" # data_dir: "/path/to/MRI_PET_"
train_batch_size: 1
train_lr: !!float 5e-4
weight_decay: !!float 1e-6
train_num_steps: 80000
save_and_sample_every: 5
num_samples: 16
gradient_accumulate_every: 1
ema_decay: 0.999
amp: false
fp16: true
calculate_fid: true
dataset: MRI2PET
image_direction: axial
num_slices: 113
ROI_mask: None 
model_cycling: false
with_DTI: false
dx_labels: ['CN', 'Dementia', 'MCI']
results_folder: "./results" 
resume: null
pretrain: null
###### evaluation config #####

test_batch_size: 32
eval_dataset: ADNI
eval_resolution: [113, 137, 113]
eval_mode: False
synthesis: true

**Comment by zzzzzj52** [comment]
I have fixed the problem by change pic size to 96
